### DISCUSSION TITLE
counters not increasing?

### DISCUSSION THREAD

**Discussion by misch42** [strongswan/strongswan#1549] [General]
I want to monitor the performance of my VPN server. So I have a look at swanctl --counters.

When I start a tunnel, the ike-init-in-req and  ike-init-out-resp increase correctly. BUT the create-child-in-req and create-child-out-resp remain zero.

Any ideas?

Version: strongswan-5.9.9-2.el8.x86_64

Michael.

**Comment by tobiasbrunner** [comment]
Did you check [what these counters actually mean](https://docs.strongswan.org/docs/5.9/plugins/counters.html)?

**Comment by misch42** [comment]
Well, no. But I looked it up now and it mean indeed the things that I thought. So no problem here.

After one hour the values of create-child-in-req and create-child-out-resp increased. So the initial tunnel setup does exchange child negotiations. Only after rekey time create-child packets are exchanged.

**Reply by tobiasbrunner** [reply]
Yes, `CREATE_CHILD_SA` exchanges are used for rekeying (IKE and CHILD_SAs) and to create CHILD_SAs. The first CHILD_SA is created during the IKE_AUTH exchange, unless childless IKE_SAs are created.
